app/addons/databases/tests/nightwatch/checkDatabaseTooltip.js

```diff
@@ -0,0 +1,49 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+module.exports = {
+  'Check the tooltip icon for DB with deleted items appears' : function (client) {
+    var waitTime = 10000,
+        newDatabaseName = client.globals.testDatabaseName,
+        newDocumentName = 'TemporaryDoc',
+        baseUrl = client.globals.test_settings.launch_url;
+
+    client
+      .loginToGUI()
+
+      // use nano to quickly set up a DB with a single doc
+      .deleteDatabase(newDatabaseName)
+      .createDatabase(newDatabaseName)
+      .createDocument(newDocumentName, newDatabaseName)
+
+      // delete the document manually. This'll ensure the database page has at least one "!" icon
+      .waitForElementPresent('#dashboard-content a[href="#/database/' + newDatabaseName + '/_all_docs"]', waitTime, false)
+      .click('#dashboard-content a[href="#/database/' + newDatabaseName + '/_all_docs"]')
+      .waitForElementVisible('label[for="checkbox-' + newDocumentName + '"]', waitTime, false)
+      .click('label[for="checkbox-' + newDocumentName + '"]')
+      .click('.control-toggle-alternative-header')
+      .waitForElementPresent('.control-select-all', waitTime, false)
+      .click('.control-delete')
+      .acceptAlert()
+      .waitForElementVisible('#global-notifications .alert.alert-info', waitTime, false)
+      .click('#nav-links a[href="#/_all_dbs"]')
+
+      // now let's look at the actual UI to confirm the tooltip appears
+      .waitForElementPresent('.js-db-graveyard', waitTime, false)
+      .execute('$(".js-db-graveyard:first").mouseenter()')
+      .pause(200)
```

Comment:
Ah, interesting! I experimented with moveToElement but it wouldn't work for me. I'll try this out.